lib/membrane/core/parent/child_life_controller/crash_group_utils.ex

```diff
@@ -30,6 +26,16 @@ defmodule Membrane.Core.Parent.ChildLifeController.CrashGroupUtils do
     )
   end
 
+  @spec extend_crash_group(
+          Child.group(),
+          ChildrenSpec.crash_group_mode(),
+          [Child.name()],
+          Parent.state()
+        ) :: Parent.state()
+  def extend_crash_group(group_name, _mode, children, state) do
```

Comment:
I would remove `mode` argument here
```suggestion
  def extend_crash_group(group_name, children, state) do
```